src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -71,7 +71,22 @@ rendering.macro.status.parameter.colour.description=The color of the status(text
 rendering.macro.status.parameter.title.name=Title
 rendering.macro.status.parameter.title.description=The text of the status. If not specified, the color name will be displayed (e.g.:"Grey").
 rendering.macro.status.parameter.subtle.name=Subtle
-rendering.macro.status.parameter.subtle.description=The aspect of the status: either white text on dark color background or dark color text on light color background. This is a boolean parameter so it should be filled with "true" or "false". Default: "false" (white text on dark color background).</content>
+rendering.macro.status.parameter.subtle.description=The aspect of the status: either white text on dark color background or dark color text on light color background. This is a boolean parameter so it should be filled with "true" or "false". Default: "false" (white text on dark color background).
+
+## Team macro
+rendering.macro.team.name=Team
+rendering.macro.team.name=description=Display avatars of the members of a team
+rendering.macro.team.parameter.tag.description=Tag that users need to have to be shown
+rendering.macro.team.parameter.size.description=The size (height and width) of an avatar
+rendering.macro.team.parameter.showUsernames.description=The name of users should be displayed under their avatars
+rendering.macro.team.parameter.letterAvatarBgColor.description=The background color of the letter avatars
+rendering.macro.team.parameter.letterAvatarFontColor.description=The font color of the letter avatars
+rendering.macro.team.parameter.disableLetterAvatars.description=Don't use letter avatars and display a default avatar instead
+rendering.macro.team.parameter.requireExternalAuth.description=The user should be searched in LDAP or OAuth
+rendering.macro.team.content.options=Options
+rendering.macro.team.content.filter=Filter…
+rendering.macro.team.content.showUsernames=Show names
+rendering.macro.team.content.noUsers=There is nobody to show.</content>
```

Comment:
Missing translation key for administration section title and description. See https://github.com/xwikisas/application-licensing/blob/master/application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/Translations.xml#L39-L40 .

Reply:
I'm lost here. When I grep that repository, I see no use of `admin.licenses.description`. What is the naming convention and how can I check it works?
Is the goal to translate <heading>Team Macro</head> in the configuration page and if so, how do I instruct XWiki to use that translation key there?

Reply:
You need to add:

```
admin.<sectionName>=Translated Section Name
admin.<sectionName>.description=Translated section descrition 
```

The key is computed by the Administration Application. You don't use this key yourself, you just have to provide the key that the Administration Application is looking for. And ``<sectionName>`` is what you specify in your configurable class object that defines the administration section.

Reply:
So that would be "Pro Macros" ? Should I rename this to something like "promacros" and put "Pro Macros" in the Translation page like this?

```suggestion
rendering.macro.team.content.noUsers=There is nobody to show.</content>
admin.promacros=Pro Macros
admin.promacros.description=Settings for Pro Macros
```